spring-cloud-gcp-samples/spring-cloud-gcp-alloydb-sample/README.adoc

```diff
@@ -0,0 +1,43 @@
+= Spring Framework on Google Cloud AlloyDB Sample
+
+This code sample demonstrates how to connect to a Google Cloud AlloyDB instance using the link:../../spring-cloud-gcp-starters/spring-cloud-gcp-starter-alloydb[Spring Framework on Google Cloud AlloyDB Starter].
+
+You will create a cluster and primary instance, a database within the instance, populate the database and then query it.
+
+== Setup
+
+image:http://gstatic.com/cloudssh/images/open-btn.svg[link=https://ssh.cloud.google.com/cloudshell/editor?cloudshell_git_repo=https%3A%2F%2Fgithub.com%2FGoogleCloudPlatform%2Fspring-cloud-gcp&cloudshell_open_in_editor=spring-cloud-gcp-samples/spring-cloud-gcp-alloydb-sample/README.adoc]
+
+1. Create a Google AlloyDB cluster and its primary instance following https://cloud.google.com/alloydb/docs/quickstart/create-and-connect[these instructions].
+
+You will be asked to set a password for the `postgres` root user; remember this value.
```

Comment:
```suggestion
  You will be asked to set a password for the `postgres` root user; remember this value.
```
Adding this indent will prevent markdown from restarting the list at `1.`, as is currently happening below this line.